### ISSUE TITLE
Logging the training loss  in sentence_prediction.py

### ISSUE THREAD

**Issue by shamanez** [pytorch/fairseq#1566]
What is the main reason behind logging the loss divided by math.log(2)?

In this [line](https://github.com/pytorch/fairseq/blob/master/fairseq/criterions/sentence_prediction.py#L88)

**Comment by lematt1991** [comment]
I believe it is to change the log from base `e` to base 2.  #1553 has more context on this.

**Comment by shamanez** [comment]
Ok got it. So when we are training a model with NLL loss, this step is there to show the loss from log base 2. But this gives wrong results when we are training the network for a regression task.
